docs/topics/images.rst

```diff
@@ -204,6 +204,29 @@ Therefore, if you'd added the field ``author`` to your AbstractImage in the abov
     The image property used for the ``src`` attribute is actually ``image.url``, not ``image.src``.
 
 
+Alternative HTML tags
+`````````````````````
+
+The `as` syntax also allows alternative HTML image tags (such as `<picture>` or `<amp-img>`) to be used.
```

Comment:
We should use double backticks for `<picture>` and `<amp-img>` here, so that they appear as monospace rather than italics.